tiledb/sm/array_schema/domain.cc

```diff
@@ -1493,8 +1494,9 @@ void Domain::compute_tile_offsets() {
   tile_offsets_col_.push_back(1);
   if (dim_num_ > 1) {
     for (unsigned int i = 1; i < dim_num_; ++i) {
-      tile_num = (domain[2 * (i - 1) + 1] - domain[2 * (i - 1)] + 1) /
-                 tile_extents[i - 1];
+      tile_num = utils::ceil(
```

Comment:
Just noting here that this will not work for `float` and `double` domains, as well as for negative domains. We will have to address this in #432 and #427, respectively.